internal/vds/metadatahandle.hpp

```diff
@@ -19,6 +19,7 @@ class MetadataHandle {
 
     BoundingBox bounding_box() const noexcept (true);
     std::string crs()          const noexcept (true);
+    std::string get_displayname() const noexcept (true);
```

Comment:
I know the above methods (`crs` and `bounding_box`) and `get_axis` below follow different conventions on using a `verb` so you fall in-between to chairs here. I'd opt for dropping get her tho, as the new method is conceptually closer to the ones above.

Reply:
I changed the function name to `input_filename`. Let me know if it follows the convention we are using or just suggest me the appropriate name.